src/execution/debug.ts

```diff
@@ -92,6 +97,17 @@ export class GaugeDebugger {
         }
     }
 
+    private updateConfigFromVscodeDebugConfig(configobject: ConfigObj) {
+        try {
+            const config = workspace.getConfiguration('launch', workspace.workspaceFolders[0].uri);
```

Comment:
there can be more than one gauge project in a workspace, and each can have their own launch.json.

instead of `workspace.workspaceFolders[0]` this should look up `workspace.workspaceFolders.find(f => f.uri.fspath === this.projectRoot)`